web/vite-plugin-prefix-classes.ts

```
@@ -0,0 +1,95 @@
+          if (className.startsWith('__') || className.startsWith('group-') || className.startsWith('peer-')) {
+            return match;
+          }
+          return `.${prefix}-${className}`;
+        }
+      );
+      
+      return {
+        code: prefixedCss,
+        map: null
+      };
+    },
```

Biome (2.1.2)

[error] This method is later overwritten by an object member with the same name.

Overwritten with this method.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this method.


(lint/suspicious/noDuplicateObjectKeys)